absl/container/internal/raw_hash_set.h

```diff
@@ -1857,6 +1859,7 @@ struct HashtableDebugAccess<Set, absl::void_t<typename Set::raw_hash_set>> {
       }
       if (g.MatchEmpty()) return num_probes;
       seq.next();
+      assert(seq.index() < capacity_ && "full table!");
```

Comment:
This doesn't build, so how was this tested?

Reply:
Not sure how I snuck that line in there, this one shouldn't be there. I will close this and resubmit.